shap/explainers/_gpu_tree.py

```diff
@@ -24,34 +24,37 @@ class GPUTreeExplainer(TreeExplainer):
+        dictated by causal inference (Janzing et al. 2019). Note that the "interventional" option
+        requires a background dataset ``data``, and its runtime scales linearly with the size of the
+        background dataset you use. Anywhere from 100 to 1000 random background samples are good
+        sizes to use.
+        The "tree_path_dependent" approach is to just follow the trees and use the number of training
+        examples that went down each leaf to represent the background distribution. This approach
+        does not require a background dataset, and so is used by default when no background dataset
+        is provided.
 
     model_output : "raw", "probability", "log_loss", or model method name
-        What output of the model should be explained. If "raw" then we explain the raw output of the
-        trees, which varies by model. For regression models "raw" is the standard output, for binary
-        classification in XGBoost this is the log odds ratio. If model_output is the name of a
-        supported prediction method on the model object then we explain the output of that model
-        method name. For example model_output="predict_proba" explains the result of calling
-        model.predict_proba. If "probability" then we explain the output of the model transformed into
-        probability space (note that this means the SHAP values now sum to the probability output of the
-        model). If "logloss" then we explain the log base e of the model loss function, so that the SHAP
-        values sum up to the log loss of the model for each sample. This is helpful for breaking
-        down model performance by feature. Currently the probability and logloss options are only
-        supported when
-        feature_dependence="independent".
+        What output of the model should be explained.
+        If "raw", then we explain the raw output of the trees, which varies by model. For regression models,
+        "raw" is the standard output. For binary classification in XGBoost, this is the log odds ratio.
+        If ``model_output`` is the name of a supported prediction method on the ``model`` object, then we
+        explain the output of that model method name. For example, ``model_output="predict_proba"``
+        explains the result of calling ``model.predict_proba``.
+        If "probability", then we explain the output of the model transformed into probability space
+        (note that this means the SHAP values now sum to the probability output of the model).
+        If "log_loss", then we explain the log base e of the model loss function, so that the SHAP values
+        sum up to the log loss of the model for each sample. This is helpful for breaking down model
+        performance by feature.
+        Currently the "probability" and "log_loss" options are only supported when
+        ``feature_perturbation="interventional"``.
```

Comment:
Nit: it could be helpful to format these in bullet points or similar, so each option is on its own line in the sphinx docs. At the moment, it seems to render as a single paragraph:

![image](https://github.com/shap/shap/assets/71127464/0e274943-2b4a-434e-8a96-c50bf06ab9f0)

Reply:
Same comment for `Tree` as for `GPUTree`

Reply:
I pushed a commit to fix this